base/applications/osk/settings.c

```diff
@@ -0,0 +1,103 @@
+﻿/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Settings file for warning dialog on startup
+ * COPYRIGHT:       Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "settings.h"
+
+/* FUNCTIONS *******************************************************************/
+
+BOOL LoadDataFromRegistry(BOOL bIsEnabled)
+{
+    HKEY hKey;
+    LONG lResult;
+    DWORD dwData;
+    DWORD cbData = sizeof(DWORD);
+
+    /* Set the structure member to the function's parameter */
+    Globals.bShowWarning = bIsEnabled;
+
+    /* Open the key, so that we can query it */
+    lResult = RegOpenKeyEx(HKEY_CURRENT_USER,
```

Comment:
OSK is inconsistent in this regard so it doesn't really matter, but for future reference, our codebase generally prefers explicit -W function names.

Reply:
Oh I thought that the compiler automatically choose the function as UNICODE (-W) because of `L""`. Thank you for pointing that out Thomas.